src/pages/Swap.tsx

```diff
@@ -287,10 +306,12 @@ function Swap(): ReactElement {
           amountToReceive = Zero
           amountMediumSynth = Zero
         } else {
+          const tokenAddr = tokens[formStateArg.from.address]?.address
+          if (!tokenAddr) return
           const [amountOutSynth, amountOutToken] =
             await bridgeContract.calcSynthToToken(
               destinationPool.metaSwapDepositAddress,
-              utils.formatBytes32String(formStateArg.from.symbol),
+              utils.formatBytes32String(tokenAddr),
```

Comment:
I believe this is incorrect - synth swapping requires the `bytes32` of the token symbol.